client/my-sites/sidebar/index.jsx

```diff
@@ -493,6 +493,30 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
+	trackHostingClick = () => {
+		this.trackMenuItemClick( 'hosting' );
+		this.onNavigate();
+	};
+
+	hosting() {
+		const { translate, path, site, isAtomicSite, siteSuffix } = this.props;
+
+		if ( ! site || ! isAtomicSite ) {
```

Comment:
We also need a capability check here and in the router, since this section won't be useful to look at for non-admins. This is okay for this PR, but let's follow up.

Alternatively we can display a reach out to your site-admin placeholder, but I'd leave that up to @apeatling or @shaunandrews to make the call on that.